src/rules/nameCasing.test.js

```diff
@@ -0,0 +1,127 @@
+import {nameCasing} from "./nameCasing";
+
+describe('nameCasing', () => {
+    it("no tables or views passed no errors", () => {
```

Comment:
It looks like your editor defaults to 4 spaces indenting. Are you running prettier?

Reply:
I use intellij so it's just the defaults. I can tweak it to whatever you prefer.

Reply:
Added an editorconfig in #14 and will reformat all these new files i added.